examples/huggingface/pytorch/text-generation/quantization/run_generation_cpu_woq.py

```diff
@@ -215,6 +216,7 @@
             scale_dtype=args.scale_dtype,
             weight_dtype=args.weight_dtype,
             layer_wise=args.layer_wise,
+            use_ipex_cpu=args.use_ipex_cpu,
```

Comment:
Synced with Haihao,
we would recommend to use 
use_ipex rather than use_ipex_cpu  

because for GPU we the one and only path is ipex as backend
so we better do not show ipex cpu here.

Reply:
improved.